pkg/cmd/experimental/bundlesecret/bundle-secret.go

```diff
@@ -0,0 +1,114 @@
+package bundlesecret
+
+import (
+	"encoding/base64"
+	"fmt"
+	kapi "github.com/GoogleCloudPlatform/kubernetes/pkg/api"
+	kcmdutil "github.com/GoogleCloudPlatform/kubernetes/pkg/kubectl/cmd/util"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/util"
+	"github.com/openshift/origin/pkg/cmd/util/clientcmd"
+	"github.com/spf13/cobra"
+	"io"
+	"io/ioutil"
+	"os"
+)
+
+const MaxSecretSize = 1 * 1024 * 1024
+const MaxFileSize = 1 * 1024 * 1024
+
+// NewCmdBundleSecret creates a new bunde-secret command. The bundle-secret command will walk a given directory
+// and load its filenames into kubernetes secret keys with values that are base64 encoded
+func NewCmdBundleSecret(f *clientcmd.Factory, parentName, name string, out io.Writer) *cobra.Command {
+
+	cmd := &cobra.Command{
+		Use:   "bundle-secret <ObjectMeta name> -f <dirname>",
+		Short: "bundle data from a directory to load kubernetes secret",
```

Comment:
thinking on this more, the command should support taking a directory or an individual file or list of files.